### DISCUSSION TITLE
Quoted Posts?

### DISCUSSION THREAD

**Discussion by forenta** [magicstone-dev/magicstone.dev#44] [Process]
Hello,
just discovered this thing here. Looks nice, good to have ya!

Has there been any discussion or decision made here yet to apply the Mastodon-patch for [quoted posts](https://gitea.treehouse.systems/treehouse/mastodon/pulls/36) or not?

**Comment by weex** [comment]
Hi @forenta and welcome. I think you're the first to bring it up. This is a general forum for Magic Stone but if you want to raise it for Ecko, it's best to create an issue in https://github.com/magicstone-dev/ecko/issues.

**Comment by forenta** [comment]
Issue opened: https://github.com/magicstone-dev/ecko/issues/477
